tests/e2e/helpers.py

```
@@ -382,6 +382,7 @@
     "automation_rename_button": "[data-testid='automation-rename-button']",
     "automation_rename_input": "[data-testid='automation-rename-input']",
     "automation_rename_save": "[data-testid='automation-rename-save']",
+    "toast": "[data-testid='toast']",
```

Ruff (0.15.21)

[error] Dictionary key literal `"toast"` repeated

Remove repeated key literal `"toast"`

(F601)